### DISCUSSION TITLE
无法发弹幕和下载视频

### DISCUSSION THREAD

**Discussion by GalaxyeyesTchyonDragon** [ywmoyue/biliuwp-lite#338] [Q&A] [answered]
重装了一遍也不行，不知道哪里有问题

**Comment by GD-Slime** [comment]
能否补充一下信息？例如复现过程

**Comment by ywmoyue** [comment]
@GalaxyeyesTchyonDragon 麻烦提供一下截图和日志，日志在设置页帮助页最下面，重装是否按照安装教程中的脚本方式安装的

**Reply by GalaxyeyesTchyonDragon** [reply]
日志发了

**Comment by GalaxyeyesTchyonDragon** [comment]
[20231008.log](https://github.com/ywmoyue/biliuwp-lite/files/12839388/20231008.log)

**Reply by GD-Slime** [reply]
我看了日志，其中有个错误是因为kgithub引起的。 这个问题在4.6.16中已经修复。请更新软件到4.6.16再试试？

**Reply by ywmoyue** [reply]
看日志是缺依赖了，请问你是使用什么方式安装的呢

**Reply by GalaxyeyesTchyonDragon** [reply]
就是照着安装说明安装的

**Reply by GalaxyeyesTchyonDragon** [reply]
![屏幕截图 2023-10-08 204500](https://github.com/ywmoyue/biliuwp-lite/assets/105159062/93572801-4ca7-454a-8e80-7fb1fca96ea7)
4.1.6装不了啊

**Reply by GD-Slime** [reply, marked as answer]
你的电脑是什么型号呢？如果cpu是intel或者amd的话需要安装x64版本的而不是arm64版本

**Reply by GalaxyeyesTchyonDragon** [reply]
可以了，谢谢
